app/views/candidate_interface/other_qualifications/review/show.html.erb

```diff
@@ -8,3 +8,12 @@
 <%= render(OtherQualificationsReviewComponent, application_form: @application_form) %>
 
 <%= govuk_button_link_to t('application_form.other_qualification.another.button'), candidate_interface_new_other_qualification_path, class: 'govuk-button--secondary' %>
+
+<%= form_with model: @application_form, url: candidate_interface_complete_other_qualifications_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+  <div class='govuk-form-group'>
+    <%= f.hidden_field :other_qualifications_completed, value: false %>
+    <%= f.govuk_check_box :other_qualifications_completed, true, multiple: false, label: { text: t('application_form.other_qualification.review.completed_checkbox') } %>
```

Comment:
I am not sure why we need a hidden field and a check_box to mantain the same value.
Am I missing something?

Reply:
This is how we have to generate a single checkbox using the formbuilder. See https://govuk-form-builder.netlify.com/form-elements/checkboxes/ and the `Generating a single checkbox` section.